backend/api/appsettings.json

```diff
@@ -1,67 +1,81 @@
 {
-  "BaseUrl": "/api",
-  "HealthChecks": {
-    "Port": "8080"
-  },
-  "HealthChecksUI": {
-    "HealthChecks": [{
-        "Name": "liveliness",
-        "Uri": "http://localhost:8080/api/health/live"
-      },
-      {
-        "Name": "readiness",
-        "Uri": "http://localhost:8080/api/health/ready"
-      }
-    ],
-    "Webhooks": [{
-      "Name": "RocketChat",
-      "Uri": "overwrite with env vars",
-      "Payload": "{\"text\":\"The HealthCheck [[LIVENESS]] is failing with the error message [[FAILURE]]. [[DESCRIPTIONS]].\"}",
-      "RestoredPayload": "{\"text\":\"The HealthCheck [[LIVENESS]] is now passing.\"}"
-    }],
-    "EvaluationTimeInSeconds": 10,
-    "MinimumSecondsBetweenFailureNotifications": 60
-  },
-  "https_port": 443,
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information",
-      "System.Net.Http.HttpClient": "Error",
-      "HealthChecks.UI": "Error"
-    }
-  },
-  "AllowedHosts": "*",
-  "Pims": {
-    "ServiceAccount": {
-      "Email": "support@pims.com"
-    }
-  },
-  "Keycloak": {
-    "Authority": "http://keycloak:8080/auth/realms/pims",
-    "Audience": "pims-api",
-    "Secret": "[USE SECRETS]",
-    "Client": "pims-api",
-    "ServiceAccount": {
-      "Audience": "pims-service-account",
-      "Secret": "[USE SECRETS]",
-      "Client": "pims-service-account"
+    "BaseUrl": "/api",
+    "HealthChecks": {
+        "Port": "8080",
+        "LivePath": "/api/health/live",
+        "ReadyPath": "/api/health/ready"
+    },
+    "HealthChecksUI": {
+        "UIPath": "/api-health/healthchecks-ui",
```

Comment:
I'd like it to be `/api-health` and create a route for this in OpenShift.  That way we can extract the UI into its own container.

Reply:
Please note that `https://pims-dev.pathfinder.gov.bc.ca/api-health/*` does not exist (currently). 
Our reverse proxy only allows root (`"/"`) and `/api`. Maybe that is why the healthcheck UI is failing to load...

Reply:
I have added `/api-health` to DEV for testing.

Reply:
On your idea if running the healthcheck UI from a separate container - I like it but maybe it will require some research. Currently that UI is being served by the ASP.NET Core web server (Kestrel). I don't know if the creators of the health-check library made it possible to extract the UI.

Reply:
In the end, having just the health-check library (without UI) seems enough to me. The UI looks nice but always felt like an add-on/nice to have. Also, there are other (free) options out there that are popular at the Lab; e.g. [Uptime Robot](https://uptimerobot.com/)